src/style.css

```diff
@@ -1,3 +1,107 @@
+html {
+  --header-height: 60px;
+  --active-link-color: rgba(0, 172, 220, 1);
+  --main-text-color: black;
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+}
+
 body {
   margin: 0;
+  padding: 0;
+}
+
+.header {
+  height: var(--header-height);
+  padding: 0 50px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.nav__list {
+  list-style: none;
+  display: flex;
+  padding: 0;
+  margin: 0;
+}
+
+.nav__link {
+  text-decoration: none;
+  color: var(--main-text-color);
+  text-transform: uppercase;
+  cursor: pointer;
+  display: flex;
+  margin-left: 20px;
+}
+
+.list__item {
+  line-height: var(--header-height);
+  position: relative;
+}
+
+.is-active {
+  display: flex;
+  flex-direction: column;
+  margin-left: 0;
+  color: var(--active-link-color);
+}
+
+.nav__link:hover {
+  color: var(--active-link-color);
+}
+
+.is-active::after {
+  content: "";
+  height: 4px;
+  background-color: var(--active-link-color);
+  border-radius: 8px;
+  position: absolute;
+  bottom: 0;
+  width: 100%;
+}
+
+.hide {
+  display: none;
+}
+
+@media all and (max-width: 900px) {
+  html {
+    font-size: 10px;
+  }
+
+  .nav__link {
+    margin-left: 10px;
+  }
+}
+
```

Comment:
Good that you are experimenting with media-queries, though it is out of scope for the current task


Hint: in real projects, you would almost never decrease the font size on a smaller screen - that will make it unreadable 😉 
No need to fix it though - it is just FYI